### PR TITLE
DBZ-3397 An exception if system language is not English

### PR DESCRIPTION
**Pull request by elgca**
https://issues.redhat.com/browse/DBZ-3397

DBZ-3397 An exception if system language is not English

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@Naros, could you take a look? Also see my comment on the Jira issue on this one. Thx!

**Review by Naros — APPROVED** [review]

**Comment by Naros** [comment]
The changes thus far are fine; however lets wait to see whats the outcome of the query I've asked to be run in the Jira issue before we merge.  If the banner text is unaffected by locale, then this is good as-is but if it is then we may need some additional tweaks.

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ private OracleDatabaseVersion resolveOracleDatabaseVersion() {
             }
             catch (SQLException e) {
                 // exception ignored
-                if (e.getMessage().contains("ORA-00904: \"BANNER_FULL\": invalid identifier")) {
+                if (e.getMessage().contains("ORA-00904")) {
```
Comment (rk3rn3r) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java:137]: Can't we use `SQLException#getSQLState()`? Is it set properly by the ODBC driver? It should contain exactly and only this error code.

Comment (Naros) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java:137]: The problem with SQL State is that we get `42000` which implies somewhat of a myriad of reasons as to why the SQL statement failed, including but not limited to insufficient permissions from what I have concluded.  If there is a situation where Oracle throws an insufficient permission problem, we should explicitly fail.  The only time we want to allow the code to move forward is explicitly if that field isn't detected.

**Comment by Naros** [comment]
Superseded by PR https://github.com/debezium/debezium/pull/2313, closing this in favor of the adjusted PR.

**Pull request closed without merging**